### DISCUSSION TITLE
split yaml to multiple files?

### DISCUSSION THREAD

**Discussion by volarname** [intlify/bundle-tools#88] [Q&A]
hello, i am looking to use yaml with following folfer structure:
```
locales/en/group1/subject1.yaml
locales/en/group1/subject2.yaml
locales/en/group2/subject3.yaml

locales/fr/group1/subject1.yaml
locales/fr/group1/subject2.yaml
locales/fr/group2/subject3.yaml
```

its possible to achieve this with `import messages from '@intlify/vite-plugin-vue-i18n/messages'` ?

**Comment by Vissie2** [comment]
Similar use-case. I'm trying to find a way to accomplish this folder structure and use the folders as language key so `messages` can understand it.

**/src/i18n/**:
```
│   index.ts
│
└───locales
    │   index.json
    │
    ├───en-GB
    │   │   global.json
    │   │
    │   ├───components
    │   │       Example.json
    │   │
    │   └───views
    └───en-US
        │   global.json
        │
        ├───components
        │       Example.json
        │
        └───views
```

Then something like `t('Example.key')` or `t('global.key')` should be available.

I thought about linking everything together in a `src/i18n/index.ts` but that is not supported either.
